### ISSUE TITLE
Remove direct email links to avoid spam

### ISSUE THREAD

**Issue by Robb-Fr** [clicepfl/clic-website#230]
As we receive a lot of mails (blocked by epfl quarantine gateway but still) to the most visible addresses (the one on the index page sponsoring.clic@epfl.ch and clic@epfl.ch), it might be cool to find a non-elegant but effective way to noise those.

As we can see on many websites, we could change sponsoring.clic@epfl.ch -> sponsoring.clic AT epfl.ch
This seems to become more and more standard

**Comment by Robb-Fr** [comment]
Yup makes sense, thanks for the answer, indeed contact form might be great but personally I don't find it very appealing, at least not as much as directly writting emails, we can keep it that way

**Comment by dialexo** [comment]
Moving this to discussions, as it may be relevant in the future
